File: 프로그래머스/0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.java
class Solution {
    public int solution(int n) {
        if(n % 7 == 0) {
            return n / 7;
        }
        else if (n % 7 != 0) {
            return n / 7 + 1;
        }
        return 0;
    }
}

File: 프로그래머스/unrated/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    if(n % 7 == 0) {
        answer = n / 7;
    }
    else if(n % 7 != 0) {
        answer = n / 7 + 1;
    }
    return answer;
}

int main() {
    int num1, result;
    scanf("%d", & num1);
    result = solution(num1);
    
    printf("%d", result);
    return 0;
}